src/component/Edit/Editproperty.jsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import Selectoption from "../../Dashboard/Selectoption";
+import axios from "axios";
+
+const url = "http://localhost:3006/v1//property/:userId";
```

Comment:
Why do you have a double slash in the URL? also you shouldn't hardcode your API URL. It should be dynamic based on the environment you are on, when you deploy this code online, your API URL will no longer be localhost:3006.

Reply:
worked on boss